Header header
float32[] x_pose
float32[] y_pose
float32 num_items